kinematics/acceleration.py

```diff
@@ -0,0 +1,129 @@
+# KINEMATICS #
```

Comment:
From here to line 24 should be a docstring instead of comments so that the content would appear in any generated documentation.